CRM/Utils/String.php

```diff
@@ -119,6 +119,31 @@ public static function convertStringToCamel($string) {
     return implode('', $fragments);
   }
 
+  /**
+   * Convert CamelCase to snake_case, with special handling for some entity names.
+   *
+   * Eg. Activity returns activity
+   *     UFGroup returns uf_group
+   *     OptionValue returns option_value
+   *
+   * @param string $entity
+   *
+   * @return string
+   */
+  public static function convertNameToLowerCase(string $entity): string {
```

Comment:
I think it needs to have the word Entity in the name since it's a special sort of name being converted

Reply:
(I kinda prefer to put it in AllCoreTables but I can live with here)

Reply:
Ok I changed the name.